backend/btrixcloud/orgs.py

```diff
@@ -443,17 +443,22 @@ async def invite_user_to_org(
         org: Organization = Depends(org_owner_dep),
         user: User = Depends(user_dep),
     ):
-        if await invites.invite_user(
+        invited_str = "new_user"
+        new_user, token = await invites.invite_user(
             invite,
             user,
             user_manager,
             org=org,
             allow_existing=True,
             headers=request.headers,
-        ):
-            return {"invited": "new_user"}
+        )
+        if not new_user:
+            invited_str = "existing_user"
 
-        return {"invited": "existing_user"}
+        return {
+            "invited": invited_str,
+            "token": token,
```

Comment:
Per [Discord convo](https://discord.com/channels/895426029194207262/910966759165657161/1072296661986062446), I think `token` should only be returned for superadmins or in a test environment, to mitigate any potential of future phishing issues

Reply:
Yeah, perhaps to be safe, we just put this in a test fixture that overrides this method and returns the token, while the default does not?
For non-org endpoint, already available to superusers only, and likely we'll remove soon.

Reply:
Heh, setting up a test fixture is not possible as it'd be a different image, so perhaps just superadmin can return token. Trying in separate branch